new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInPV.java

```diff
@@ -0,0 +1,625 @@
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Tests to create domain in persistent volume using WLST.
+ */
+@DisplayName("Verify istio enabled WebLogic domain in domainhome-on-pv model")
+@IntegrationTest
+public class ItIstioDomainInPV  {
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  private static String image = WLS_BASE_IMAGE_NAME + ":" + WLS_BASE_IMAGE_TAG;
+  private static boolean isUseSecret = true;
+
+  private final String wlSecretName = "weblogic-credentials";
+  private final String domainUid = "istio-div";
+  private final String clusterName = "mycluster";
+  private final String adminServerName = "admin-server";
+  private final String adminServerPodName = domainUid + "-" + adminServerName;
+  private static LoggingFacade logger = null;
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+
+  /**
+   * Assigns unique namespaces for operator and domains.
+   * Pull WebLogic image if running tests in Kind cluster.
+   * Installs operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+
+    logger = getLogger();
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Assign a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    domainNamespace = namespaces.get(1);
+
+    // Label the operator/domain namespace with istio-injection=enabled
+    Map<String, String> labelMap = new HashMap();
+    labelMap.put("istio-injection", "enabled");
+
+    assertDoesNotThrow(() -> addLabelsToNamespace(domainNamespace,labelMap));
+    assertDoesNotThrow(() -> addLabelsToNamespace(opNamespace,labelMap));
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    //determine if the tests are running in Kind cluster. if true use images from Kind registry
+    if (KIND_REPO != null) {
+      String kindRepoImage = KIND_REPO + image.substring(TestConstants.OCR_REGISTRY.length() + 1);
+      logger.info("Using image {0}", kindRepoImage);
+      image = kindRepoImage;
+      isUseSecret = false;
+    }
+  }
+
+  /**
+   * Create a WebLogic domain using WLST in a persistent volume using wdt.
```

Comment:
It says both WLST and wdt, fix the description.

Reply:
done